cmd/ingress.go

```diff
@@ -17,6 +17,10 @@ func (a *MethodK8s) InitIngressCommand() {
 		Short: "Enumerate Ingresses",
 		Long:  `Enumerate Ingresses`,
 		Run: func(cmd *cobra.Command, args []string) {
+			// Init Cmd Failed
```

Comment:
when is it ever possible that a.OutputSignal.ErrorMessage is not Nil at this point in the code path?